templates/docs/examples/patterns/tabs/content-dark.html

```diff
@@ -0,0 +1,58 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Tabs / Tabbed Content Dark{% endblock %}
+
+{% block standalone_css %}patterns_tabs{% endblock %}
+
+{% block content %}
+<div class="p-strip--dark">
```

Comment:
This doesn't work in standalone example (because it doesn't have styles for dark strips).

https://vanilla-framework-4817.demos.haus/docs/examples/standalone/patterns/tabs/content-dark

I think it's easiest to hardcode the background colour in this case (we do it in other places):

```suggestion
<div class="p-strip--dark" style="background: #262626;">
```